### ISSUE TITLE
시니어 사용자가 어려워하는 UX 5가지

### ISSUE THREAD

**Issue by uraflower** [FEDeepDive/DaliyPost#193]
[시니어 사용자가 어려워하는 UX 5가지](https://toss.tech/article/senior-usability-research)

토스 유저 리서치팀에서 시니어 사용자의 사용성을 분석한 글입니다.
시니어 사용자를 다루고 있긴 하지만 나이가 많은 세대라 어려워하는 것은 아니고 익숙하냐 아니냐, 아냐 모르냐의 차이인 것 같아요.
그래서 읽을수록 신기했습니다. 평소에 쓰던 UI/UX 자체가 완전 바람직한 디자인(구조?)는 아니었던 걸지도 모르겠다는 생각이 들었거든요. 많이 쓰고 또 보편적으로 쓰이니까 사실은 명확한 구조가 아닌데도 모르고 사용했던 걸수도 있겠네 하고 가볍게 생각해봤습니다.

+)
오늘이 12월 30일이니까 내일이 올해의 마지막 날이네요!
다들 올 한 해 수고 많으셨고 다음 년도에도 잘 부탁드리옵니다 🙇 ❤️

**Comment by Young2un** [comment]
아니 3주 연속 미림님 글이라니!!! 이번주는 무조건 올리겠습니다!! 다정한 말도 넘 좋네요 앞으로도 잘 부탁드립니다🍀
